### PR TITLE
Refactor content on Choosing your courses page

### PR DESCRIPTION
**Pull request by george-bit**
## Context

Update of content as per new content design

## Changes proposed in this pull request

Before:
image

After:
image

## Guidance to review

Review against design template: https://docs.google.com/document/d/1zjQYNBrEggefjuwdBtNoCH7k9xeZlHAQ8I1CSvqwH8I/edit#

## Link to Trello card

https://trello.com/c/Prml0X76/1841-dev-content-changes-on-choosing-your-courses-page

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by EmmaFrith** [comment]
Thanks @george-bit !

Just noticed a few things improve on the content - see here: 

https://docs.google.com/document/d/1zjQYNBrEggefjuwdBtNoCH7k9xeZlHAQ8I1CSvqwH8I/edit?usp=sharing

Sorry and thanks :))

**Comment by george-bit** [comment]
@EmmaFrith - No worries I've amended based on the new design. Please review.

**Comment by EmmaFrith** [comment]
thanks @george-bit - perfect!

File: app/views/candidate_interface/application_choices/_guidance.html.erb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<p class="govuk-body">You can apply for up to 3 courses at this stage of your application.</p>
-<p class="govuk-body">Not all courses and training providers are signed up to this service. If you choose a course that isn’t signed up to <%= service_name %>, you’ll be directed to UCAS to continue your application.</p>
-<p class="govuk-body">You can preview <%= govuk_link_to 'courses currently available', candidate_interface_providers_path %> on <%= service_name %>.</p>
-<p class="govuk-body">Get support by emailing <%= govuk_link_to 'becomingateacher@digital.education.gov.uk', 'mailto:becomingateacher@digital.education.gov.uk' %>.</p>
+<p class="govuk-body">You can apply for up to 3 courses.</p>
+<p class="govuk-body">Some courses are not on <%= service_name %> yet and are only available through UCAS Teacher Training.</p>
+<p class="govuk-body"><%= govuk_link_to 'Check which courses you can apply to on ' + service_name, candidate_interface_providers_path %></p>
```
Comment (davidgisbey) [inline app/views/candidate_interface/application_choices/_guidance.html.erb:3]:
```suggestion
<p class="govuk-body"><%= govuk_link_to 'Check which courses you can apply to on ' + service_name, candidate_interface_providers_path %>.</p>
```

Should this have a full stop? I'm actually not sure haha

Comment (tvararu) [reply app/views/candidate_interface/application_choices/_guidance.html.erb:3]: I think not

Comment (george-bit) [reply app/views/candidate_interface/application_choices/_guidance.html.erb:3]: I did think but was kinda following the design so wasn't sure will update now and then 🚢

Comment (george-bit) [reply app/views/candidate_interface/application_choices/_guidance.html.erb:3]: Just to clarify I haven't added the full stop, my bad for the confusion

Comment (EmmaFrith) [reply app/views/candidate_interface/application_choices/_guidance.html.erb:3]: Hey @george-bit @tvararu @davidgisbey, my rule of thumb is if the whole sentence is hyperlinked and it's an instruction, such as 'Learn more.....' 'Check.....', 'Go to....' - don't use a full stop. That's based on what I've seen elsewhere on gov.uk

**Pull request merged**